docs/content/latest/api/ysql/datatypes/type_array/array-constructor.md

```diff
@@ -0,0 +1,193 @@
+---
+title: The array[] value constructor
+linkTitle: array[] constructor
+headerTitle: The array[] value constructor
+description: The array[] value constructor
+menu:
+  latest:
+    identifier: array-constructor
+    parent: api-ysql-datatypes-array
+    weight: 10
+isTocNested: false
+showAsideToc: false
+---
+
+The `array[]` value constructor is a special variadic function. Uniquely among all the functions described in the present major _"[Array data types and functionality](../../type_array/)"_ section, it uses square brackets to surround its list of actual arguments.
+
+## Purpose and signature
+
+**Purpose:** create an array value from scratch using an expression for each of the array's values. Such an expression can itself use the `array[]` constructor or an _[array literal](../literals/)_. 
+
+**Signature** 
+```
+input value:       [anyarray | [ anyelement, [anyelement]* ]
+return value:      anyarray
+```
+**Notes:** You can meet the goal of creating an array from directly specified values, instead, by using an _[array literal](../literals/)_.
+
+These thee ordinary functions also create an array value from scratch:
+
+- The _[array_fill()](../functions-operators/array-fill/)_ builtin SQL function creates a "blank canvas" array of the specified shape with all values set the same to what you want.
+- The _[array_agg()](../functions-operators/array-agg-unnest/#array-agg)_ builtin SQL function creates an array (of, in general, an implied _"row"_ type) from a SQL subquery.
```

Comment:
```suggestion
- The _[array_agg()](../functions-operators/array-agg-unnest/#array-agg)_ built-in SQL function creates an array (of, in general, an implied _"row"_ type) from a SQL subquery.
```

Reply:
I'm leaning towards using "predefined" instead of "built-in," but need to think a bit more about it.

Reply:
This one (change "builtin" to built-in" which I did globally before the push that you watched me do) has brought the "outdated" status and the "commit" button is disabled. My only option seems to be to ignore it. There's no way for me to say that it's already fixed.

Reply:
So I'm clicking "Resolve conversation".